platform-api/src/routers/forms.py

```
@@ -0,0 +1,124 @@
+from typing import List
```

Ruff (0.15.12)

[warning] `typing.List` is deprecated, use `list` instead

(UP035)